build/root/.bazelrc

```diff
@@ -40,19 +40,18 @@ build:cross:darwin_amd64 --platforms=@io_bazel_rules_go//go/toolchain:darwin_amd
 build:cross:windows_386 --platforms=@io_bazel_rules_go//go/toolchain:windows_386
 build:cross:windows_amd64 --platforms=@io_bazel_rules_go//go/toolchain:windows_amd64
 
-# We enable cgo cross-compilation for Linux, but need to use our custom crosstool.
-build:repo_infra_crosstool --crosstool_top=@io_k8s_repo_infra//tools:toolchain --compiler=gcc
-build:cross:linux_386 --config=repo_infra_crosstool --platforms=@io_bazel_rules_go//go/toolchain:linux_386
-build:cross:linux_amd64 --config=repo_infra_crosstool --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 --cpu=amd64
-build:cross:linux_arm --config=repo_infra_crosstool --platforms=@io_bazel_rules_go//go/toolchain:linux_arm --cpu=arm
-build:cross:linux_arm64 --config=repo_infra_crosstool --platforms=@io_bazel_rules_go//go/toolchain:linux_arm64 --cpu=arm64
-build:cross:linux_ppc64le --config=repo_infra_crosstool --platforms=@io_bazel_rules_go//go/toolchain:linux_ppc64le --cpu=ppc64le
-build:cross:linux_s390x --config=repo_infra_crosstool --platforms=@io_bazel_rules_go//go/toolchain:linux_s390x --cpu=s390x
+# These cross-compilation targets are most likely non-functional.
```

Comment:
@ixdy should we remove this ...?

Reply:
I guess it depends on whether we think anyone will fix it.

Reply:
This was all based on CROSSTOOL files where are no longer supported as of 1.0.0. `@io_k8s_repo_infra//toosl:toolchain` is already long gone. My understanding is that no body is using these right now.

https://github.com/bazelbuild/bazel/issues/5380
https://docs.google.com/document/d/1Nqf16jqDGWSrPp4VuRxh0iNnVBoAXsO0meDH69J9xoc/edit

Reply:
right, someone would need to rewrite the `CROSSTOOL` file I wrote in Starlark.

Reply:
It's seems reasonable to skip for now given that it's not used, significant effort to port, and not as high of a priority as finishing the version bump.

Reply:
💯 for doing what is needed and effective, not getting too lost in the weeds

Reply:
er not asking to do any of that in this PR, just noticing that given this doesn't work we should consider cleaning it up later.

Reply:
I filed https://github.com/kubernetes/repo-infra/issues/181 to track.